substrate/frame/election-provider-multi-block/src/signed/tests.rs

```diff
@@ -345,33 +340,20 @@ mod calls {
 			// account 1 submitted nothing, so bail should have no effect and return error
 			assert_noop!(
 				SignedPallet::bail(RuntimeOrigin::signed(bailing_account_id)),
-				SubmissionNotRegistered::<Runtime>
-			);
-		})
-	}
-
-	#[test]
-	fn force_clear_submission_fails_if_called_by_account_none() {
-		ExtBuilder::default().build_and_execute(|| {
-			assert_err!(
-				SignedPallet::force_clear_submission(RuntimeOrigin::none(), 0, 99),
-				BadOrigin
+				Error::<T>::SubmissionNotRegistered
 			);
 		})
 	}
 
 	#[test]
 	fn force_clear_submission_fails_if_called_in_phase_other_than_off() {
 		ExtBuilder::default().build_and_execute(|| {
-			let some_bn = 0;
```

Comment:
doesn't need to be a separate variable, otherwise the developer will think this is important value to the tests.